test/integration/integration_test.go

```diff
@@ -455,6 +477,7 @@ func executingOcCommandSucceedsOrFails(command string, expectedResult string) er
 }
 
 func executingMinishiftCommandSucceedsOrFails(command string, expectedResult string) error {
+	fmt.Println(command)
```

Comment:
Do you want to have this here permanently? If so, maybe we should format it a bit more